test/fs.bats

```diff
@@ -206,9 +206,12 @@ teardown() {
     [[ "$output" == linking* ]] || false
 }
 
+@test "fs.is_ellipsis_symlink should detect symlink pointing back to ELLIPSIS_PACKAGES" {
+    skip "No test implementation"
+}
 
-@test "fs.is_ellipsis_symlink should detect symlink pointing back to ELLIPSIS_PATH" {
```

Comment:
This test should still work in it's original form, otherwise the implementation got "broken" or at least is not in line with the current behavior.

Reply:
Fixed. I took the initiative to extend a bit the tests, at least to illustrate how works those functions. Also, I don't feel the need to document them, as they looks like more low level functions, and that would only result to clutter the API documentation.